PHILOSOPHY.md

```diff
@@ -0,0 +1,158 @@
+## Design goals
+
+The design goals for Tendermint (and the SDK and related libraries) are:
+
+ * Simplicity and Legibility
```

Comment:
Could you elaborate more with some explicit examples? I'm confused about how Amino and IAVL fit into simplicity? Where does using external libraries instead of rewriting ourselves fall into simplicity?